packages/web-react/.eslintrc

```diff
@@ -0,0 +1,60 @@
+{
+  "root": true,
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "parser": "@typescript-eslint/parser", // the TypeScript parser we installed earlier
+  "parserOptions": {
+    "project": "./config/tsconfig.json",
+    "ecmaFeatures": { "jsx": true } // Allows for the parsing of JSX
+  },
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "extensions": [".js", ".jsx", ".ts", ".tsx"]
+      }
```

Comment:
js and jsx are not needed here anymore?

Reply:
Yes, I've removed them.

Reply:
`js` is in base config na `jsx` is not used anywhere.